.travis.yml

```diff
@@ -33,26 +41,35 @@ jobs:
       script:
         - make release.npm crate=medea-jason publish=no
 
-    - name: medea (stable)
+    - name: medea docker (stable)
       stage: build
       rust: stable
-      script: cargo build --bin medea
+      script: make docker.build.medea debug=no TAG=${TRAVIS_BUILD_ID} registry=quay.io
```

Comment:
Also, it's better to apply `build-${TRAVIS_BUILD_ID}` Docker tag.

Raw numbers are misleading.